### PR TITLE
IBM Spectrum Scale Ansible NFS Role pull request 

### PR DESCRIPTION
**Pull request by shubh1410**
This ansible role performs two tasks:
1. Ensures that all the prerequisites required to enable CES are satisfied and then enable and configure CES.
2. Ensure all the prerequisites required to enable NFS are satisfied , install the packages required by NFS protocol and then enable the NFS protocol.

### CODE REVIEW

File: roles/nfs/precheck/tasks/storage.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    - nfs is defined
+
+    with_items:
+
+    - "{{ nfs }}"
+
+      # - debug:
+
+      # msg: "{{ nfs_params }}"
+
```
Comment (rajan-mis) [inline roles/nfs/precheck/tasks/storage.yml]: @shubh1410  We are not using storage.yml file , so please remove this file from commit.

File: roles/nfs/Readme.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+  
+ 
+Prerequisite
+----------------------------
+- CES should be enabled and configured.
+- Atleast 1 node should be designated as nfs node.
+- Native nfs service should be stopped, before usinf this role.
+- nfs-kernel-server service should be stopped.
+- knfs-server service should be stopped
+- The packages required for nfs installation are as follows:
```
Comment (rajan-mis) [inline roles/nfs/Readme.md]: @shubh1410 Please add here ansible playbook will automatically install this package .

File: roles/nfs/precheck/tasks/storage.yml (hunk 1 above)
Comment (shubh1410) [reply roles/nfs/precheck/tasks/storage.yml]: @rajan-mis suggested changes implemented

File: roles/nfs/Readme.md (hunk 2 above)
Comment (shubh1410) [reply roles/nfs/Readme.md]: @rajan-mis suggested changes implemented

File: roles/nfs/Readme.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+Implementation
+-------------------------
+- `Precheck`
+  - This role creates a list of nfs nodes and checks if atleast one node is set as nfs node.
+  - It checks if all the above defined commands are stopped before running this role. If a service is found running, the execution of this role fails.
+  
+- `Node`
+  - Installation via local package: rpms are on local machine
+    - File name: install_local_pkg.yml
+    - Path: /root/ibm-spectrum-scale-install-infra/roles/nfs/node/tasks
```
Comment (rajan-mis) [inline roles/nfs/Readme.md]: @shubh1410 Please remove /root/ from here.

Comment (shubh1410) [reply roles/nfs/Readme.md]: @rajan-mis suggested changes implemented

**Comment by rajan-mis** [comment]
Currently we are not exposing this in the main README. We are going to validate with complete development branch code and then will expose support in the main README.  i am going to merge this code .

**Comment by rajan-mis** [comment]
@whowutwut i think now git config looks good . can you please check this once then we can merge this code into dev branch. thanks

**Comment by rajan-mis** [comment]
@whowutwut I have shared multiple review comments and i am working with him for this functionality from day 1.   Can you please check GIT config setup if this is fine now. thanks

**Pull request closed without merging**